graphql/handler/apollofederatedtracingv1/tracing.go

```diff
@@ -8,18 +8,35 @@ import (
 	"google.golang.org/protobuf/proto"
 
 	"github.com/99designs/gqlgen/graphql"
+	"github.com/vektah/gqlparser/v2/gqlerror"
+)
+
+const (
+	ERROR_MASKED     = "masked"
+	ERROR_UNMODIFIED = "all"
+	ERROR_TRANSFORM  = "transform"
 )
 
 type (
 	Tracer struct {
 		ClientName string
 		Version    string
 		Hostname   string
+		Errors     TraceErrors
```

Comment:
```suggestion
		ErrorOptions     *TraceErrorOptions
```
This should be a pointer, and the name should not be easily confused with the actual errors.